### ISSUE TITLE
better table formatting

### ISSUE THREAD

**Issue by jeremymanning** [ContextLab/leetcode-solutions#31]
Our [table](https://github.com/ContextLab/leetcode-solutions#problems-weve-attempted-so-far) of attempted problems is starting to get unwieldy.  It'd be nice to have some other way of displaying it!

**Comment by jeremymanning** [comment]
suggested approach: we can modify update_readme.py to add in `<details>` tags around each month's problems, with only the most recent month "open" by default.

when the update readme action runs, we do the following:
 - if it's the first day of the month:
   - find the last `<details open>` tag in the table and replace it with `<details>`
   - add a new `<details open>` tag after the final `</details>` closing tag in the table
   - add a new `<summary>` line (`<summary><b>Problems from Month, Year</b></summary>`, replacing "Month" and "Year" with the spelled-out month name and 4-digit year, respectively)
   - add the table header lines for this month
   - add an additional `</details>` closing tag to the end of the table
   
 - add the new problem's table entry before the final `</details>` closing tag in the table

In other words, we want each month's problems to be in a separate table enclosed in `<details>...</details>` tags, with the most recent month's details set to "`open`".

we will also need to manually update the existing table by manually adding in details tags at the appropriate positions.

**Comment by jeremymanning** [comment]
More info about syntax: https://gist.github.com/pierrejoubert73/902cc94d79424356a8d20be2b382e1ab

**Comment by jeremymanning** [comment]
Addressed in [71c1fdb](https://github.com/ContextLab/leetcode-solutions/commit/71c1fdbbf052e092ec7a9f4bc7cc4b6d4d887067)!
